docs/network.md

```diff
@@ -0,0 +1,217 @@
+# 4. Code flow - finding first peer to connect to.
+Each node maintains list of peers.
+
+New node will get initial list of peers from:
+- Genesis config `network.boot_nodes`
```

Comment:
Genesis config is not the right term here. This is in client config (config.json)